README.md

```diff
@@ -9,21 +9,42 @@ Fleet Telemetry is a server reference implementation for Tesla's telemetry proto
 
 The service handles device connectivity as well as receiving and storing transmitted data. Once configured, devices establish a WebSocket connection to push configurable telemetry records. Fleet Telemetry provides clients with ack, error, or rate limit responses.
 
-## Configuring and running the service
+## Prerequisites
+There are a few system prerequisites/recommendations for installing and deploying a Fleet Telemetry server. It is highly recommended that you run this service in a Debian server. In order for any Fleet Telemetry server to interact with vehicles, you will need to own a website domain and host Fleet Telemetry on that domain, which is defined as the `partner_domain` in the [Fleet API documentation](https://developer.tesla.com/docs/fleet-api#fleet-telemetry).
+
+### Minimum System Requirements
+These minimum system requirements for any Fleet Telemetry server with 10 active vehicles connected running on a Debian-based server with regular day-to-day usage. Note that the requirements will be much higher depending on vehicle usage or number of vehicles.
+
+* 0.2 AWS vCPU
+* 256MB of RAM
+* 8GB of Storage
 
-As a service provider, you will need to register a publicly available endpoint to receive device connections. Tesla devices will rely on a mutual TLS (mTLS) WebSocket to create a connection with the backend. The application has been designed to operate on top of Kubernetes, but you can run it as a standalone binary if you prefer.
+### Dependencies
+Fleet Telemetry has the following dependencies:
 
-### Install on Kubernetes with Helm Chart (recommended)
-For ease of installation and operation, we recommend running Fleet Telemetry on Kubernetes. Helm Charts help you define, install, and upgrade applications on Kubernetes. You can find a reference helm chart [here](https://github.com/teslamotors/helm-charts/blob/main/charts/fleet-telemetry/README.md).
+* Go 1.20+
+* libzmq
+* Docker*
+* Kubernetes/Helm*
 
-### Install manually (skip this if you have installed with Helm on Kubernetes)
-1. Allocate and assign a [FQDN](https://en.wikipedia.org/wiki/Fully_qualified_domain_name). This will be used in the server and client (vehicle) configuration.
+*Docker and Kubernetes/Helm are required for installing through deploying an image or using Helm Charts (recommended).
 
-2. Design a simple hosting architecture. We recommend: Firewall/Loadbalancer -> Fleet Telemetry -> Kafka.
+### Vehicle Compatibility
+
+Vehicles must be running firmware version 2023.20.6 or later. You may find this information under your VIN in the Tesla Mobile App or the Software tab in the vehicle's infotainment system. Some older model S/X are not supported.
```

Comment:
@aaronpkahn we might want to drop a line here about the new self-service version.